tools/terraform/networking.tf

```diff
@@ -67,7 +67,13 @@ resource "google_compute_url_map" "global" {
   description     = "a description"
   default_service = google_compute_backend_service.global.id
 
+
   // Create a host rule to match traffic to alias (gcpping.com)
+  host_rule {
+    hosts        = ["*"]
```

Comment:
If we say * here, does the one for other hosts still work? Could you add some tests?

Reply:
Yes, the redirect for the alias domain `gcpping.com` (two `p`s) will still work. 

The way the URL maps are evaluated is: 
- match the most specific host rule, then match the most specific path rule within that host
- else if no host is matched, go to the `*` host rule that matches the most specific path rule
- else go to the default backend

I added tests for the url maps of the primary domain `gcping.com`, but could not add tests for the domain alias `gcpping.com` (two `p`s). 

Terraform `test` for `google_compute_url_map` ([doc](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_url_map#test)) doesn't support redirects (see https://github.com/hashicorp/terraform-provider-google/issues/9536). 

(PS: found and fixed redirect related bug for the alias domain, see the other comment I added)